### ISSUE TITLE
{Linked:Bug9651748;Linked:Bug9651685} [Screen Readers-MyHub Viva UX (Report It Now Feature)-Click on Submit button.]: Screen Readers are not announcing the Role of "Success" dialog box.

### ISSUE THREAD

**Issue by Raisul123** [microsoft/AdaptiveCards#8195]
### Target Platforms

Other

### SDK Version

https://adaptivecards.io/ 

### Application Name

Adaptive Cards

### Problem Description

Test Environment:

OS: Windows 11 Enterprise 21H2

Browser: New Edge Version 100.0.1185.44 (Official build) (64-bit)

URL: [Dashboard (sharepoint-df.com)](https://microsoft.sharepoint-df.com/teams/dev_myhubring0/SitePages/Dashboard.aspx?xsdata=MDV8MDF8fDMxZDUwOTBlMjQ2YzRiMmZiYjA0MDhkYTQyMmI5MTQ5fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDB8MHw2Mzc4OTUwNTIxMTQ5MDE3NjR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxNVGs2TUdVME5qQTFaV1psTWpkaE5HTTBOamt3TVRWaU4yRXhZV014WXprMU9XVkFkR2h5WldGa0xuWXl8fA%3D%3D&sdata=WFJWVzJNeHJpT2o1cm9MdHdUK3NnOWxzSjlScVYrSWFteFhCd0Z3eW5Nbz0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-shajain%40microsoft.com)

Repro Steps:
Step 1: Navigate to the above URL in the latest Edge Chromium with valid credentials.
Step 2: Navigate to-> Misplaced Badge->Fill Details ->Click on Submit button.
Step 3: Observe whether Screen Readers are announcing the role of "Success" dialog box. or not

Actual Result:
Screen Readers are not announcing the Role of "Success" dialog box.

Expected Result:
Screen Readers should announce the Role of "Success" dialog box.

PWD Impact:

As "Screen Readers are not announcing the 'the Role of "Success" dialog box. it will create confusion for any screen reader dependent users. The users relying on screen reader would be unable to guess the correct functionality of dialog box. which will prevent the visually disabled users to perform their essential task.

Note:
Issue is replicated throughout the application wherever the dialog box is present.

MAS Reference:

[MAS 4.1.2 – Name, Role, Value](https://aka.ms/MAS4.1.2)

### Screenshots

https://user-images.githubusercontent.com/23114460/209965711-01f06071-5727-4d48-a8af-207949efb568.mp4

https://user-images.githubusercontent.com/23114460/209965726-ab361aef-a299-4857-99c7-e7054468dacc.mp4

### Card JSON

```JSON
{
	"schema": "http=//adaptivecards.io/schemas/adaptive-card.json",
	"type": "AdaptiveCard",
	"version": "1.2",
	"body": [
		{
			"id": "",
			"isVisible": true,
			"separator": false,
			"spacing": "padding",
			"type": "Container",
			"items": [
				{
					"id": "",
					"isVisible": true,
					"separator": false,
					"spacing": "default",
					"type": "Container",
					"items": [],
					"verticalContentAlignment": "center",
					"bleed": false,
					"minHeight": "25px"
				},
				{
					"id": "",
					"isVisible": true,
					"separator": false,
					"spacing": "padding",
					"type": "Container",
					"items": [
						{
							"id": "",
							"isVisible": true,
							"separator": false,
							"spacing": "default",
							"type": "Image",
							"height": "auto",
							"horizontalAlignment": "center",
							"size": "auto",
							"style": "default",
							"url": "https://microsoft.sharepoint.com/sites/catalog/ClientSideAssets/12345678-9f17-4b50-8e1f-ea3145d8e95f/check_79d2d387fb7b90a4950cf7e7a26fe364.png",
							"altText": "Check Image"
						},
						{
							"id": "",
							"isVisible": true,
							"separator": false,
							"spacing": "default",
							"type": "TextBlock",
							"textStyle": "default",
							"wrap": true,
							"text": "Thank You",
							"size": "extraLarge",
							"weight": "bolder",
							"horizontalAlignment": "center",
							"color": "good"
						},
						{
							"id": "",
							"isVisible": true,
							"separator": false,
							"spacing": "default",
							"type": "TextBlock",
							"textStyle": "default",
							"wrap": true,
							"text": "Your incident has been reported.\n You will receive a follow up mail soon.",
							"size": "default",
							"color": "dark",
							"horizontalAlignment": "center"
						}
					],
					"verticalContentAlignment": "center",
					"bleed": false
				},
				{
					"id": "",
					"isVisible": true,
					"separator": false,
					"spacing": "default",
					"type": "Container",
					"items": [],
					"verticalContentAlignment": "center",
					"bleed": false,
					"minHeight": "25px"
				},
				{
					"id": "",
					"isVisible": true,
					"separator": false,
					"spacing": "default",
					"type": "ActionSet",
					"actions": [
						{
							"style": "positive",
							"id": "HomeView",
							"title": "Submit Another Incident",
							"altText": "Submit Another Incident",
							"type": "Action.Submit",
							"associatedInputs": "auto",
							"data": { "id": "HomeView" }
						}
					]
				}
			],
			"verticalContentAlignment": "center",
			"bleed": false
		}
	],
	"actions": [],
	"refresh": null
}
```

**Comment by Raisul123** [comment]
Since we are not able to create the UI using the provided Json and we got the confirmation from the Viva connection team [https://onedrive.visualstudio.com/WEX!/_workitems/edit/1505534](https://onedrive.visualstudio.com/WEX!/_workitems/edit/1505534) that the issue belongs to Adaptive Card team hence we are created the bug using the external attachments and bug details

**Comment by anna-dingler** [comment]
Hi @Raisul123,

Can you provide anymore context for this issue? It is not reproducible.

When I navigate to the test site and report a misplaced badge, a success dialog does not appear. Is there an updated link that has the correct workflow?

Thanks,
Anna

**Comment by anna-dingler** [comment]
Closing since this bug does not repro with the provided URL.

**Comment by vagpt** [comment]
Closing this bug as per the above comment.
